src/CachePlugin.php

```diff
@@ -183,11 +236,12 @@ protected function isCacheable(ResponseInterface $response)
         if (!in_array($response->getStatusCode(), [200, 203, 300, 301, 302, 404, 410])) {
             return false;
         }
-        if (!$this->config['respect_cache_headers']) {
```

Comment:
On line 361 I've added some BC logic. If this option is false we add an empty array so all the checks are skipped and original behavior is kept